lib/DDG/Meta/Data.pm

```diff
@@ -161,6 +161,13 @@ sub get_ia {
     return clone($m);
 }
 
+sub get_js {
+    my ($self, $id) = @_;
+
+    my $metaj = encode_json($self->get_ia(id => $id));                                                                                                                                                                                              
+	return qq(DDH.$id = DDH.$id || {};\nDDH.$id.meta = $metaj;); 
```

Comment:
Yeah, weird, spaces at the end of the line.  I usually catch those with git diff.